src/sentry/grouping/api.py

```diff
@@ -61,7 +61,7 @@ class GroupingConfigNotFound(LookupError):
 
 class GroupingConfig(TypedDict):
     id: str
-    enhancements: Enhancements
+    enhancements: str
```

Comment:
I'm surprised about this change.

Reply:
My guess is: It was passing because `_get_enhancements` didn't had any types for the return value.